src/components/CardComponent.jsx

```diff
@@ -0,0 +1,19 @@
+const CardComponent = ({items}) => {
```

Comment:
can destructure this further. something like this 
const CardComponent = ({items:{name, location}})=>(
<div>

</div>
);